<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerticalActuatorWidget</class>
 <widget class="QWidget" name="VerticalActuatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VerticalActuatorWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="foo_0">
     <item>
      <widget class="QLabel" name="foo_device_label">
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DeviceSelectionBtn"/>
     </item>
     <item>
      <widget class="QPushButton" name="DeviceSetBtn">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="foo_1">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSpinBox" name="StepSizeSpinBox">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MoveUpBtn">
       <property name="text">
        <string>↑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MoveDownBtn">
       <property name="text">
        <string>↓</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="foo_2">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="KpInput">
         <property name="value">
          <double>10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="foo_labelkp">
         <property name="text">
          <string>Kp</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="KiInput">
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="foo_labelki">
         <property name="text">
          <string>Ki</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="KdInput">
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="foo_labelkd">
         <property name="text">
          <string>Kd</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="ILimInput">
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="foo_labelilim">
         <property name="text">
          <string>I_Limit</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="OutLimInput">
         <property name="value">
          <double>100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="foo_labeloutlim">
         <property name="text">
          <string>Out_Limit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="SetPIDBtn">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLCDNumber" name="PositionLCD"/>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="TargetForceSpinBox">
     <property name="value">
      <double>10.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="StartForceBtn">
     <property name="text">
      <string>Start Force Control</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
